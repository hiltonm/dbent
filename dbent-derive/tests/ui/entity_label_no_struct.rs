use dbent::prelude::*;

#[derive(Label)]
enum Test {
    Id(Key<Int>),
    #[label] Data(String),
}

fn main() {}
