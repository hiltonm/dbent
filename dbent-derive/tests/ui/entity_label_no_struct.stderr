error: proc-macro derive panicked
 --> tests/ui/entity_label_no_struct.rs:3:10
  |
3 | #[derive(Label)]
  |          ^^^^^
  |
  = help: message: #[derive(Label)] can only be used on structs
